app/helpers/app_form_builder.rb

```diff
@@ -52,7 +52,7 @@ def radio_button(method, tag_value, options = {})
   end
 
   def radios(method, text, options = {}, &block)
-    radios_div = @template.content_tag(:div, class: 'govuk_radios') do
+    radios_div = @template.content_tag(:div, class: 'govuk_radios', id: "#{@object.model_name.singular}_#{method}") do
```

Comment:
Prida id na radio group, aby error link mohol linkovat na ne.

Reply:
`dom_id` poznas?

Reply:
Nie, pozriem to :)

Reply:
Pozeram ze `dom_id` view hodit len prefix, ale nie suffix takze satle musim tam nieco resit manualne.
Vyzeralo by to nejake takto: `"#{ActionView::RecordIdentifier.dom_id(@object)}_#{method}"` takze neviem ci to dava uplne zmysel.

Reply:
Chapem, nechajme takto.